dev/Pager/PrototypePager/Scripts/Int32MinusOneConverter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.UI.Xaml.Data;
+
+namespace MUXControlsTestApp.Scripts
+{
+    class Int32MinusOneConverter : IValueConverter
```

Comment:
>Int32MinusOneConverter [](start = 10, length = 22)

Personally I don't think we should make a Minus one converter.  I guess we could maybe think about a PlusX converter with a parameter and we could pass -1. But this is moving towards an expression converter.  @Ranjeshj and @mikehillberg thoughts?  I think by convension this would be a template setting

Reply:
I will remove this converter but I'm still interested in hearing your thoughts on this!

I made the converter because I thought it helped simplify the flow of the control since I wouldn't need to create a duplicate variable that is just offset by 1